app/src/features/surveys/components/StudyAreaForm.tsx

```diff
@@ -124,6 +157,31 @@ const StudyAreaForm = () => {
         draw_controls_ref={drawRef}
       />
 
+      {errors.locations && !Array.isArray(errors?.locations) && (
+        <Box pt={2}>
+          <Typography style={{ fontSize: '12px', color: '#f44336' }}>{errors.locations}</Typography>
```

Comment:
Is it possible to use `sx={{ color: 'error' }}`?

Reply:
this has since been updated to an alert with the severity set to 'error'